rerun/src/main/java/org/apache/falcon/rerun/handler/LateRerunHandler.java

```diff
@@ -62,7 +62,7 @@ public void handleRerun(String cluster, String entityType, String entityName, St
         try {
             Entity entity = EntityUtil.getEntity(entityType, entityName);
             int intRunId = Integer.parseInt(runId);
-            Date msgInsertTime = EntityUtil.parseDateUTC(nominalTime);
+            Date msgInsertTime = new Date(msgReceivedTime);
```

Comment:
Thanks @pallavi-rao for pointing out.
Closed this pull request and raised another pull request with only a single commit.